packages/front-end/components/Billing/StripeProviderWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import { createContext, useContext, useState } from "react";
+
+interface StripeContextProps {
+  clientSecret: string | null;
+  setClientSecret: (secret: string | null) => void;
+}
+
+const StripeContext = createContext<StripeContextProps | undefined>(undefined);
+
+export const StripeProviderWrapper = ({
```

Comment:
It's better to move this logic into `StripeProvider` so when using this, you only have a single wrapping component.  With how it currently is, someone might forget to add one or the other and it would cause weird errors that would be hard to debug.